src/biotite/sequence/graphics/alignment.py

```diff
@@ -344,7 +344,142 @@ def get_color(self, alignment, column_i, seq_i):
+    recognition data obtaine from microarrays. Symbols are visualized as characters 
+    on a colored background box. The color of a given box represents the recognition 
+    signal. The intensity of the color, is proportional to the strenght of the 
+    recognition.
+    
+    Parameters
+    ----------
+    axes : Axes
+        A Matplotlib axes, that is used as plotting area.
+    fl_score : numpy.ndarray 
+        The ndarray to map recognition values to score residues.
+        By default the normalized score is 1 for maximun recognition
+        and 0 for non-recognition(no color).
+    color_symbols : bool, optional
+        If true, the symbols themselves are colored.
+        If false, the symbols are black, and the boxes behind the
+        symbols are colored.
+    font_size : float, optional
+        Font size of the sequence symbols.
+    font_param : dict, optional
+        Additional parameters that is given to the
+        matplotlib.Text instance of each symbol.
+    
+    '''
+    def __init__(self, axes, fl_score, color_symbols=False,
+                 font_size=None, font_param=None):
+
+        super().__init__(axes, color_symbols, font_size, font_param)
+        if fl_score is not None:
+            self.fl_score = fl_score 
+        else:
+            self.fl_score = None        
+        self._cmap = self._generate_colormap(colors["dimorange"],
+                                             self._color_symbols) 
+             
+                             
+    def get_color(self, alignment, column_i, seq_i):
+        '''
+        Get the color of a symbol at a specified position in the
+        alignment.
+        The symbol is specified as position in the alignment's trace
+        (trace[pos_i, seq_i]).
+
+        Parameters
+        ----------
+        alignment : Alignment
+            The respective alignment.
+        column_i : int
+            The position index in the trace.
+        seq_i : int
+            The sequence index in the trace.
+        Returns
+        -------
+        color : object
+            A *Matplotlib* compatible color used for the background
+            or at the specifed position
+            '''
+        index1 = alignment.trace[column_i, seq_i]
+        if index1 == -1:
+            spot_signal = 0
+        else:
+            spot_signal = self._get_signal(self.fl_score, column_i, seq_i)
+        return self._cmap(spot_signal)
+
+
+    def _get_signal(self, fl_score, column_i, seq_i):
+        if fl_score is None:
+            signal = 0.01
+        else:
+            signal = fl_score[column_i, seq_i]
+        return signal
+
+    # getter for cmap:
+    def get_cmap(self):
+        return self._cmap
 
+           
+    def plot_symbol(self, bbox, alignment, column_i, seq_i):
+        '''
+        This method is optimized to draw the symbols of an alignment while
+        mapping the molecular recognition data throughout the entire sequence 
+        alignment.
+        '''
```

Comment:
The parameter description section is missing from this docstring. You can just adapt it from the parent class. Alternatively, you could remove this docstring.

Reply:
I will remove the docstring.